server/utils/dateUtils.ts

```diff
@@ -119,6 +119,20 @@ export class DateFormats {
       [`${key}-year`]: `${date.getFullYear()}`,
     } as ObjectWithDateParts<K>
   }
+
+  static isoDateToDaysFromNow(dateString: string) {
+    const difference = differenceInDays(new Date(dateString).setHours(0, 0, 0, 0), new Date().setHours(0, 0, 0, 0))
```

Comment:
`date-fns`'s `differenceInDays` returns the number of complete 24hr cycles between dates: this means it finds 0 days between `2024-04-08T12:00:00` and `2024-04-09T00:00:00`. To avoid this pitfall, we set both compared dates to `T00:00:00.000`.